.github/workflows/verify.yml

```
@@ -380,3 +389,24 @@ jobs:
       - name: Compare files
         run: | #shell
           diff --strip-trailing-cr --recursive --color=always source package
+
+      - name: Detect Akismet version
+        if: env.PACKAGE_INCLUDES_AKISMET == 'true'
+        run: | #shell
+          AKISMET_VERSION="$(grep -E '^\s*\*?\s*Version:' akismet-package/akismet.php | head -n1 | sed -E 's/^.*Version:[[:space:]]*([0-9.]+).*$/\1/')"
```

Juggle with one ball only.
```suggestion
          AKISMET_VERSION="$(sed -nE 's/^[[:space:]]*\*?[[:space:]]*Version:[[:space:]]*([0-9.]+).*$/\1/p;q' akismet-package/akismet.php)"
```

---

@johnbillion These workflow runs bury this comment.